### ISSUE TITLE
Move : Staticfiles to `./static/`

### ISSUE THREAD

**Issue by Aryamanz29** [sruti2024/PRO_Act#164]
-  Some static files are present in` ./templates/ directory `

-  Move it to `./static/` and change their links in the codebase if necessary.

### **SS 👇:**

![Screenshot (85)](https://user-images.githubusercontent.com/56113566/115932259-35445b80-a4aa-11eb-885b-cd4a8c0798bb.png)

**Comment by Sin-Sumit** [comment]
@SaiSudhaV  Can I work on this? Please assign, if yes.

**Comment by Aryamanz29** [comment]
> @SaiSudhaV  Can I work on this? Please assign, if yes.

Hello 👋
Just raised PR for this issue .. No need to ask for assignment.

**Comment by SaiSudhaV** [comment]
> > @SaiSudhaV  Can I work on this? Please assign, if yes.
> 
> Hello 👋
> Just raised PR for this issue .. No need to ask for assignment.

Yeah you can start working and raise the PR...need not ask or wait for the assignment

**Comment by Sin-Sumit** [comment]
Hey@SaiSudhaV, While installing the requirements black version is not correct because that it didn't install that and on migrating the manage.py file, it shows an error for module dotenv, but on installing that dotenv separately, it is not downloading.

**Comment by Sin-Sumit** [comment]
And I checked the file structure where I found that there are no files that need to be in the static folder. All are good at their places.
